test/execution/test_protocol.py

```diff
@@ -178,6 +178,29 @@ def protocol(alice, bob):
     assert "remote_get" in (action.name for action in traced_actions)
 
 
+def test_trace_store_load():
+    @sy.func2protocol(roles=["alice", "bob"], args_shape={"alice": ((1,),), "bob": ((1,),)})
+    def protocol(alice, bob):
+        t = th.Tensor([101])
+        stored = bob.store(t)
```

Comment:
Still feel like I should mention that I was thinking of `load` as something that happens at the beginning and `store` as something that happens at the end. There's no reason it _has_ to be that way I suppose, but I'd love to see at least one test that does that. I think it will become increasingly important to test as @Jasopaum makes progress on `Executions`.

Reply:
added to the same test that loads then stores.